code/modules/clothing/chameleon.dm

```diff
@@ -161,7 +161,7 @@
 			I.sprite_sheets = P.sprite_sheets
 			qdel(P)
 
-		if(istype(I, /obj/item/clothing) && istype(initial(picked_item), /obj/item/clothing))
+		if(istype(I, /obj/item/clothing) && istype(picked_item, /obj/item/clothing))
 			var/obj/item/clothing/CL = I
 			var/obj/item/clothing/PCL = picked_item
 			CL.flags_cover = initial(PCL.flags_cover)
```

Comment:
Forgot to mention it but kube noticed this bit of code doesn't work (with or without `initial()` because `picked_item` is a typepath being passed to `istype()`, not an instance.

Reply:
`/datum/action/item_action/chameleon/change/proc/update_item(obj/item/picked_item)`

`istype(picked_item, /obj/item/clothing))` 

Seems fine?

Reply:
And here's a link to the ref in case you still think this would work: http://www.byond.com/docs/ref/#/proc/istype

Reply:
Why the heck are we passing a typepath as an object

Reply:
Because it doesn't need to be instantiated.